docs/reference/contrib/settings.rst

```diff
@@ -296,8 +296,8 @@ You could write:
 
 .. code-block:: html+django
 
-    {{ settings.app_label.ImportantPages.page_url.donate_page }}
-    {{ settings.app_label.ImportantPages.page_url.sign_up_page }}
+    {{ settings.app_label.ImportantPages.donate_page.url }}
+    {{ settings.app_label.ImportantPages.sign_up_page.url }}
 
 Using the ``page_url`` shortcut has a few of advantages over using the tag:
```

Comment:
Should this line also be updated, along with the description underneath, it refers to the comparison between `page_url` the tag and the attribute/method on the `Page`